libs/discord/discord.cpp

```diff
@@ -0,0 +1,212 @@
+
+//#include "qcommon/qcommon.h"
+#include "cgame/cg_local.h"
+#include "windows/miniwindows.h"
+#include <time.h>
+#include "discord_game_sdk.h"
+
+#define CLIENT_ID 882369979406753812
```

Comment:
This is a testing application tied to my account.
You can register your game at https://discord.com/developers/
Preferably add it to a team account: https://discord.com/developers/docs/topics/teams so that multiple people can manage it.
Artwork (shown in the previews) is among the things where you need this for.

Reply:
I renamed the testing application to `Cocaine Diesel` and gave you access, feel free to add whoever you want and update artwork.